platform/ios/Makefile

```diff
@@ -68,6 +74,19 @@ upload_ipa: export_ipa
 		--apiKey $(appstore_api_key_id) \
 		--apiIssuer $(appstore_api_issuer_id)
 
+.PHONY: release_to_external_testers
+release_to_external_testers:
+	cd $(app_store_connect_cmd); \
+	swift run AppStoreConnectCMD \
+	 release-to-external-testers \
+	 $(appstore_api_issuer_id)\
+     $(appstore_api_key_id) \
```

Comment:
Nit: Weird indentation

Reply:
Weird, it was not showing in Sublime.
Fixed, somehow.